inst/rmarkdown/templates/lipics/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,302 @@
+
+# Typesetting instructions -- Summary
+
+LIPIcs is a series of open access high-quality conference proceedings across all fields in informatics established in cooperation with Schloss Dagstuhl.
+In order to do justice to the high scientific quality of the conferences that publish their proceedings in the LIPIcs series, which is ensured by the thorough review process of the respective events, we believe that LIPIcs proceedings must have an attractive and consistent layout matching the standard of the series.
+Moreover, the quality of the metadata, the typesetting and the layout must also meet the requirements of other external parties such as indexing service, DOI registry, funding agencies, among others. The guidelines contained in this document serve as the baseline for the authors, editors, and the publisher to create documents that meet as many different requirements as possible.
+
+Please comply with the following instructions when preparing your article for a LIPIcs proceedings volume.
+
+## Minimum requirements {.unnumbered}
+
+- Use pdflatex and an up-to-date \LaTeX{} system.
+- Use further \LaTeX{} packages and custom made macros carefully and only if required.
+- Use the provided sectioning macros: `\section` (`#` in Markdown), `\subsection` (`##` in Markdown), `\subsubsection` (`###` in Markdown), 
+  `\paragraph`, `\paragraph*`, and `\subparagraph*`.
+-  Provide suitable graphics of at least 300dpi (preferably in PDF format).
+-  Use BibTeX and keep the standard style (`plainurl`) for the bibliography.
+-  Please try to keep the warnings log as small as possible. Avoid overfull `\hboxes` and any kind of warnings/errors with the referenced BibTeX entries.
+-  Use a spellchecker to correct typos.
+
+## Mandatory metadata macros {.unnumbered}
+
+**NOTE**: _The following macros are set via the YAML header in the R Markdown template._
+
+Please set the values of the metadata macros carefully since the information parsed from these macros will be passed to publication servers, catalogues and search engines.
+Avoid placing macros inside the metadata macros. The following metadata macros/environments are mandatory:
+
+- `\title` and, in case of long titles, `\titlerunning`.
+- `\author`, one for each author, even if two or more authors have the same affiliation.
+- `\authorrunning` and `\Copyright` (concatenated author names)\\
+The `\author` macros and the `\Copyright` macro should contain full author names (especially with regard to the first name), while `\authorrunning` should contain abbreviated first names.
+- `\ccsdesc` (ACM classification, see \url{https://www.acm.org/publications/class-2012}).
+- `\keywords` (a comma-separated list of keywords).
+- `\relatedversion` (if there is a related version, typically the ``full version''); please make sure to provide a persistent URL, e.\,g., at arXiv.
+- `\begin{abstract}...\end{abstract}`.
+
+\paragraph*{Please do not \ldots} <!-- Do not override the \texttt{\seriesstyle}-defaults} -->
```

Comment:
I believe you can write this also as markdown
```md
#### Please do not \dots {.unumbered}
```
```suggestion
#### Please do not \ldots {.unnumbered}
<!-- Do not override the \texttt{\seriesstyle}-defaults} -->
```
not what you want ? Just asking in case there is a reason